MBBSEmu/Btrieve/BtrieveFile.cs

```diff
@@ -65,6 +64,28 @@ public ushort RecordLength
             }
         }
 
+        private ushort _physicalRecordLength;
+        /// <summary>
+        ///     Actual Length of the records within the Btrieve File, including additional padding.
+        /// </summary>
+        public ushort PhysicalRecordLength
```

Comment:
Should we [JsonIgnore] this since it's only needed on initial load?

Reply:
I'm inclined to keep it since it serves as useful debugging information when you look at the `.EMU` file